src/_pytest/doctest.py

```diff
@@ -331,14 +352,16 @@ def collect(self):
         optionflags = get_optionflags(self)
 
         runner = _get_runner(
-            verbose=0,
+            verbose=False,
             optionflags=optionflags,
             checker=_get_checker(),
             continue_on_failure=_get_continue_on_failure(self.config),
         )
 
         parser = doctest.DocTestParser()
-        test = parser.get_doctest(text, globs, name, filename, 0)
+        # Remove ignore once this reaches mypy:
+        # https://github.com/python/typeshed/commit/3e4a251b2b6da6bb43137acf5abf81ecfa7ba8ee
+        test = parser.get_doctest(text, globs, name, filename, 0)  # type: ignore
```

Comment:
Seems to be in v0.740.

Reply:
I'll run `--warn-unused-ignores` sometime, for now I'll keep to ease rebasing and such.

Reply:
Sure, just thought it might make sense to remove it when rebasing after v0.740, which can be merged soon already.
It makes also sense to rebase after #6039 (just merged) then to ensure the test still passes.

Reply:
OK, will do.